Userland/Libraries/LibTest/JavaScriptTestRunnerMain.cpp

```diff
@@ -109,6 +110,7 @@ int main(int argc, char** argv)
         },
     });
     args_parser.add_option(print_json, "Show results as JSON", "json", 'j');
+    args_parser.add_option(detailed_json, "Show detailed results as JSON (implies -j)", "detailed-json", 0);
```

Comment:
Maybe this could be named `per-file` as well for symmetry with the test262 runner?
Definitely not needed though.

Reply:
:thinking: 
Alright, will switch to that, sounds fine to me.